src/Domain/Apprenticeship/ApprenticeshipEpisode.cs

```diff
@@ -80,12 +68,12 @@ public void Update(Apprenticeships.Types.ApprenticeshipEpisode episodeUpdate)
         UKPRN = episodeUpdate.Ukprn;
     }
 
-    private void UpdateEarningsProfile(ApprenticeshipFunding.ApprenticeshipFunding apprenticeshipFunding, List<Earning> earnings, ISystemClockService? systemClock)
+    private void UpdateEarningsProfile(IEnumerable<Earning> earnings, ISystemClockService? systemClock, decimal onProgramTotal, decimal completionPayment)
```

Comment:
I don't think its possible for systemclock to be null

Reply:
There were 2 different methods that called this, calculate and recalculate. Calculate I think intentionally passed in a null and recal didn't